### ISSUE TITLE
eventio.EventIOFile init takes forever when trying to open a gzipped file

### ISSUE THREAD

**Issue by dneise** [fact-project/pyeventio#33]
I just tried to open this file 

```python
eventio.EventIOFile(
'gamma_20deg_0deg_run4601___cta-prod3-merged_desert-2150m-Paranal-3HB89-NGFD_cone10.simtel.gz')
```

That took forever .. I killed it before it ended. not sure if this is due to the well known slow python unzipping or if I just tried to binary read a zipped file :-D

gotta look into it.

**Comment by MaxNoe** [comment]
Try unzipping , ihr will only take two seconds . If I remember correctly, it took about 90 seconds gzipped. 

This is because it parses the whole file and a lot of forward backward seeking is done, which is always slow with gzip.

I want to take care of this by generating the containers, not parsing the whole file by default.

**Comment by dneise** [comment]
I totally agree here.

**Comment by MaxNoe** [comment]
solved by #35
